index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <title>Microverse Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="navigation">
+            <div>
+                <a  class="logo" href="#">My Logo</a>
+            </div>
+            <div class="menu-placeholder">
+                <a href="#"><img src="./images/Union.svg" alt="menu icon"></a>
+            </div>
+        </nav>
+    </header>
+    <div class="first-container">
+        <div class="Hello">
+            <h1 class="introduction">I'm Ivan<br>Glad to see you</h1>
+        </div>
+        <div>
+            <p class="who-am-i">I’m a software developer! I can help you build a product, feature or website. Look through some of my work and
+            experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a href="#" class="connect">Let's connect</a>
+        </div>
+        <div>
+            <ul class="social">
+                <li class="social-icon"><a href="#" class="twitter"><img src="./images/twitter.png" alt="twitter icon"></a></li>
+                <li class="social-icon"><a href="#" class="linkedin"><img src="./images/linkedin.png" alt="linkedin icon"></a></li>
+                <li class="social-icon"><a href="#" class="medium"><img src="./images/medium.png" alt="medium icon"></a></li>
+                <li class="social-icon"><a href="#" class="github"><img src="./images/github.png" alt="github icon"></a></li>
+                <li class="social-icon"><a href="#" class="angellist"><img src="./images/angellist.png" alt="angellist icon"></a></li>
```

Comment:
- [ ] For the main body section, please enclose the HTML code using the `<section>` tag. This is in order to enhance the semantics of your HTML code.

Reply:
Ok, done! Thank you!